keras_nlp/models/bert/bert_tokenizer.py

```diff
@@ -113,51 +111,9 @@ def __init__(
     def presets(cls):
         return copy.deepcopy({**backbone_presets, **classifier_presets})
 
-    @classmethod
-    @format_docstring(names=PRESET_NAMES)
     def from_preset(
```

Comment:
After following the changes below re docstrings and `__init__subclass__` you should be able to remove the `from_preset` method here and elsewhere entirely!

Reply:
@shivance, we don't need `from_preset` in subclasses anymore! See, for example, [BertPreprocessor](https://github.com/keras-team/keras-nlp/blob/fb1eeb9054a2a13be4853a6e635e9a72d54d04db/keras_nlp/models/bert/bert_preprocessor.py#L195)